rslib/src/scheduler/states/load_balancer.rs

```diff
@@ -389,7 +390,7 @@ pub fn select_weighted_interval(
     Some(intervals_and_weights[selected_interval_index].0)
 }
 
-fn interval_to_weekday(interval: u32, next_day_at: TimestampSecs) -> usize {
+pub fn interval_to_weekday(interval: u32, next_day_at: TimestampSecs) -> usize {
```

Comment:
Personally, I prefer to use pub(crate) in such circumstances, even if the entire file is already pub(crate), as it keeps things explicit, and won't break if the module is made public in the future for some reason.